src/components/Utilites/TickChange.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-underscore-dangle */
+const TickChange = () => {
+  const ticksFound = document.querySelectorAll('.tick > text');
+  const tickLinesFound = document.querySelectorAll('.tick > line');
+  const tickLinesFoundX = document.querySelectorAll(
+    '.d3-chart__datesX > .tick > line'
+  );
+  const DomainsFound = document.querySelectorAll('.domain');
+  // D3 Chart
+  // D3 Chart
+  // D3 Chart
+  for (let i = 0; i < tickLinesFoundX.length; i += 1) {
+    tickLinesFoundX[i].style.display = 'none';
+  }
+  //   Making the X-AXIS DISAPPEAR
+  DomainsFound[0].style.display = 'none';
+  // Changes Numbers to Percent
+  for (let i = 0; i < ticksFound.length; i += 1) {
+    const tickInnerHtml = ticksFound[i].innerHTML;
+    // Removes numbers of Y-axis on 10,30,50,70 and 90.
+    if (tickInnerHtml === '10') {
+      ticksFound[i].style.display = 'none';
+    } else if (tickInnerHtml === '30') {
+      ticksFound[i].style.display = 'none';
+    } else if (tickInnerHtml === '50') {
+      ticksFound[i].style.display = 'none';
+    } else if (tickInnerHtml === '70') {
+      ticksFound[i].style.display = 'none';
+    } else if (tickInnerHtml === '90') {
+      ticksFound[i].style.display = 'none';
+    }
+
+    // Changes numbers of Y-axis on 20,40,60,80 and 100 to %.
+
+    if (tickInnerHtml === '20') {
+      ticksFound[i].innerHTML = '20%';
+    } else if (tickInnerHtml === '40') {
+      ticksFound[i].innerHTML = '40%';
+    } else if (tickInnerHtml === '60') {
+      ticksFound[i].innerHTML = '60%';
+    } else if (tickInnerHtml === '80') {
+      ticksFound[i].innerHTML = '80%';
+    } else if (tickInnerHtml === '100') {
+      ticksFound[i].innerHTML = '100%';
+    } else if (tickInnerHtml === '0') {
+      ticksFound[i].innerHTML = '0%';
+    }
```

Comment:
One way that might work is to do the above is to take a modulus division of 20 to see if there's a remainder. If yes, set `ticksFound[i].style.display = 'none';` else simply append the percentage sign of `tickInnerHtml`.

```
if (tickInnerHtml % 20 > 0) {
ticksFound[i].style.display = 'none';
} else {
ticksFound[i].innerHTML = `${tickInnerHtml}%`;
}
```

Reply:
Perfect, let me put implement these changes and push.